File: Lasagna.js
const PREPARATION_MINUTES_PER_LAYER = 2;
export const EXPECTED_MINUTES_IN_OVEN = 40;

export function remainingMinutesInOven(actualMinutesInOven) {
  return EXPECTED_MINUTES_IN_OVEN - actualMinutesInOven;
}

export function preparationTimeInMinutes(numberOfLayers) {
  return PREPARATION_MINUTES_PER_LAYER * numberOfLayers;
}

export function totalTimeInMinutes(numberOfLayers, actualMinutesInOven) {
  return preparationTimeInMinutes(numberOfLayers) + actualMinutesInOven;
}

// Sample usage for testing:
console.log(remainingMinutesInOven(30)); // 10
console.log(preparationTimeInMinutes(2)); // 4
console.log(totalTimeInMinutes(3, 20)); // 26

File: lasagna.py
EXPECTED_BAKE_TIME=40

def bake_time_remaining(port):
    """Bake time remaining 
    calculation"""
    return EXPECTED_BAKE_TIME-port
    
def preparation_time_in_minutes(number_of_layers):
    """Preparation time in minutes calculation"""
    return number_of_layers*2
    
def elapsed_time_in_minutes(number_of_layers,elapsed_bake_time):
    """Elapsed time in minutes calculation
    """
    return number_of_layers*2+elapsed_bake_time

#Sample usage for testing
print(bake_time_remaining(10))  # 30
print(preparation_time_in_minutes(2))  # 4
print(elapsed_time_in_minutes(3, 20))  # 26